src/Libraries/RevitNodes/Elements/ModelCurve.cs

```diff
@@ -134,20 +131,35 @@ private void InitModelCurve(Autodesk.Revit.DB.Curve crv, bool makeReferenceCurve
         #region Private mutators
 
         /// <summary>
-        /// Set the curve internally.  Returns false if this method failed to set the curve
+        /// Set the plane and the curve internally.
         /// </summary>
-        /// <param name="c"></param>
-        /// <returns></returns>
-        private bool InternalSetSketchPlaneFromCurve(Autodesk.Revit.DB.Curve c)
+        private void InternalSetSketchPlaneFromCurve(Curve newCurve)
         {
             TransactionManager.Instance.EnsureInTransaction(Document);
 
-            // Infer the sketch plane
-            Autodesk.Revit.DB.Plane plane = CurveUtils.GetPlaneFromCurve(c, false);
+            Plane newPlane = CurveUtils.GetPlaneFromCurve(newCurve, false);
+            Plane oldPlane = InternalCurveElement.SketchPlane.GetPlane();
+
+            var angleBetweenPlanes = newPlane.Normal.AngleTo(oldPlane.Normal);
```

Comment:
API says it's radians.
How can we check it?
I mean... Should I check whether it's radians or not?
![image](https://cloud.githubusercontent.com/assets/8158404/13882784/9c09e1c4-ed2f-11e5-924a-a872ecba58bc.png)